server/apps/main/templates/main/about_us.html

```diff
@@ -22,15 +22,17 @@
           <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
              href="#about-autspaces">About AutSPACEs<span class="badge badge-pill"></span></a>
           <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
-             href="#platform-usage">How to use the Platform<span class="badge badge-pill"></span></a>
+             href="#platform-usage">How to use the AutSPACEs<span class="badge badge-pill"></span></a>
```

Comment:
href="#platform-usage">How to use AutSPACEs<span class="badge badge-pill"></span></a>

Reply:
(extra "the" in there)